lib/posix-sysinfo/sysinfo.c

```diff
@@ -41,6 +41,11 @@
 #include <sys/sysinfo.h>
 #include <uk/syscall.h>
 
+#ifdef CONFIG_PAGING
+#include <uk/plat/paging.h>
+#include <uk/falloc.h>
+#endif
```

Comment:
Close conditional passages with a comment on what `#ifdef` their refer to, since we do not indent macros.

```suggestion
#ifdef CONFIG_HAVE_PAGING
#include <uk/plat/paging.h>
#include <uk/falloc.h>
#endif /* CONFIG_HAVE_PAGING */
```